File: 프로그래머스/lv0/120860. 직사각형 넓이 구하기/직사각형 넓이 구하기.py
def solution(dots):
    g,s = 0, 0
    # 첫번째 좌표와, 그것과 x좌표가 같은 좌표를 찾아 세로 길이 구하기
    # 첫번째 좌표와 그것과 y좌표가 같은 좌표를 찾아 가로 길이 구하기
    for i in dots:
        if dots[0][0]==i[0]:
            s = abs(dots[0][1]-i[1])
        if dots[0][1] ==i[1]:
            g = abs(dots[0][0]-i[0])
    return s*g

File: 프로그래머스/lv0/120860. 직사각형 넓이 구하기/직사각형 넓이 구하기.js
function solution(dots) {
    //첫번째 좌표와 x좌표 같은거 찾아서 높이, y좌표 같은 거 찾아서 너비
    var a = dots[0];
    var h, w = 0;
    for (var k = 1;k<dots.length;k++){
        if (dots[k][0]===a[0]){
            console.log()
            h = Math.abs(a[1]-dots[k][1]);
        }
        if (dots[k][1]===a[1]){
            w = Math.abs(a[0]-dots[k][0]);
        }
    }
    return w*h;
}